src/components/tooltip/resourcesview.js

```diff
@@ -0,0 +1,71 @@
+import styled from 'styled-components';
+import {productionResources, calculateEachBuildingEnergy} from '../../util/ResourcenProduction';
+import {useSelector} from 'react-redux';
+
+export default function ResourcesView({isOpen, currentRess}) {
+  const currentUserBuildings = useSelector(state => state.currentUserBuildings);
+  const resNameUpperCase = currentRess ? currentRess.name.charAt(0).toUpperCase() + currentRess.name.slice(1) : '';
+
+  let currentHourProduction = 0;
+  let currentEnergyProduction = 0;
+  let currentEnergyUsed = 0;
+
+  if (currentRess) {
+    if (currentRess.name !== 'gold') {
+      const currentTotalEnergy = calculateEachBuildingEnergy(currentUserBuildings);
+
+      currentTotalEnergy.forEach(energy => {
+        if (energy < 0) {
+          currentEnergyUsed += energy;
+        } else {
+          currentEnergyProduction += energy;
+        }
+      });
+
+      currentHourProduction = productionResources(currentRess.name, currentUserBuildings);
+    }
+  }
+
+  return (
+    <>
+      <StyledSection isOpen={isOpen}>
+        <p>
+          <strong>{resNameUpperCase}</strong>
+        </p>
+        <hr />
+        <StyledP>
+          <span>You have</span>
+          <span>{Math.floor(currentRess.value)}</span>
+        </StyledP>
+        <StyledP>
+          <span>{currentRess.name === 'energy' ? 'Current Production' : 'Current /h Production'}</span>
+          <span>{currentRess.name === 'energy' ? currentEnergyProduction : currentHourProduction}</span>
+        </StyledP>
+        <StyledP>
+          <span>{currentRess.name === 'energy' ? 'Current using' : ''}</span>
+          <span>{currentRess.name === 'energy' ? currentEnergyUsed : ''}</span>
+        </StyledP>
+      </StyledSection>
+    </>
```

Comment:
Please have a look at my previous comments regarding the fragment. As well, I would recommend to replace the p elements (StyledP) with another element (e.g. div). It looks like you use these only as a wrapper (so for styling purposes). A paragraph on the other hand is a block of text... 

```suggestion
      <StyledSection isOpen={isOpen}>
        <p>
          <strong>{resNameUpperCase}</strong>
        </p>
        <hr />
        <StyledP>
          <span>You have</span>
          <span>{Math.floor(currentRess.value)}</span>
        </StyledP>
        <StyledP>
          <span>{currentRess.name === 'energy' ? 'Current Production' : 'Current /h Production'}</span>
          <span>{currentRess.name === 'energy' ? currentEnergyProduction : currentHourProduction}</span>
        </StyledP>
        <StyledP>
          <span>{currentRess.name === 'energy' ? 'Current using' : ''}</span>
          <span>{currentRess.name === 'energy' ? currentEnergyUsed : ''}</span>
        </StyledP>
      </StyledSection>
```